### PR TITLE
[CLOUD-1600] chore: snyk code scanning

### PR DESCRIPTION
**Pull request by craigfurman**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -191,8 +209,13 @@ workflows:
           filters:
             branches:
               ignore: [main, gh-pages]
-      - "Snyk scan":
-          context: ["snyk-apps"]
+      - snyk-deps:
+          context: ["tiki-snyk"]
```
Comment (craigfurman) [inline .circleci/config.yml:213]: @tommyknows do you remember why we used this other context before? As it stands now I've made it similar to kubernetes-resource-store.

**Comment by craigfurman** [comment]
Screenshot 2023-08-03 at 15 10 35

Weird, this is a renamed job

File: .circleci/config.yml (hunk 1 above)
Comment (tommyknows) [reply .circleci/config.yml:213]: I probably copy-pasted this from somewhere else. IIRC in container we've used that snyk-apps context everywhere (not sure if we still do though). 

I didn't know that we had a separate context for it, TIL.

**Review by tommyknows — APPROVED** [review]

**Comment by craigfurman** [comment]
I think this PR is just borked (circle check that can never possibly return). I'll open a new one

**Pull request closed without merging**